javascript/app-discover/src/api/api.ts

```diff
@@ -0,0 +1,74 @@
+/*!
+ * Copyright (c) Microsoft. All rights reserved.
+ * Licensed under the MIT license. See LICENSE file in the project.
+ */
+import { sleep } from '../utils/Sleep.js'
+import type {
+	DiscoverProgressCallback,
+	DiscoverStartResponse,
+	DiscoverStatusResponse,
+} from './types.js'
+import { DiscoverResponseStatus } from './types.js'
+
+const RUN_CAUSAL_DISCOVERY_BASE_URL = process.env.DISCOVER_API_URL?.endsWith(
```

Comment:
not sure about this env var adjustment. What if someone configures their environment with '/api/discover/'

Reply:
This might be some legacy code that I just moved from another file, I think it is adjusting to always have it ending with `/`, I will rewrite this with something more elegant.